### DISCUSSION TITLE
use as a generator

### DISCUSSION THREAD

**Discussion by omar-dulaimi** [MichalLytek/typegraphql-nestjs#29] [Q&A] [answered]
Hello,

Is there any way to generate resolvers and emit them out, like the other repo `typegraphql-prisma` does?

So is this something doable currently? If not, any plans for the future?

**Comment by MichalLytek** [comment, marked as answer]
No, `typegraphql-nestjs` is a package that allows you to use TypeGraphQL resolvers in NestJS app.
So you can use the resolvers generated by `typegraphql-prisma` in your NestJS app.
